ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/util/DataStructureUtil.java

```diff
@@ -2153,6 +2156,31 @@ public List<SignedBlobSidecarOld> randomSignedBlobSidecarsForBlock(
         .collect(toList());
   }
 
+  public List<BlobSidecar> randomBlobSidecarsForBlock(final SignedBeaconBlock block) {
+    return randomBlobSidecarsForBlock(block, (__, builder) -> builder);
+  }
+
+  public List<BlobSidecar> randomBlobSidecarsForBlock(
+      final SignedBeaconBlock block,
+      final BiFunction<Integer, RandomBlobSidecarBuilder, RandomBlobSidecarBuilder> modifier) {
```

Comment:
Better rename `modifier` to `blobSidecarBuilder`

Reply:
I think `blobSidecarBuilderModifier` would be clear. It's generally a BuilderBuilder :)